src/hud_elements.cpp

```diff
@@ -536,7 +545,7 @@ void HudElements::graphs(){
             arr.erase(arr.begin());
         }
 
-        HUDElements.max = gpu_info.memoryTotal;
+        HUDElements.max = gpu_info[0].memoryTotal;
```

Comment:
This seems like a typo from me? I assume that shouldn't be vram, it should be system ram

Reply:
I've pushed a fix for that. It is the total VRAM. The fix builds VRAM graphs for all GPUs